package_metrics/metrics.py

```diff
@@ -67,22 +68,32 @@ def main():
         help="package manager to calculate metrics for",
         choices=["pip"]
     )
-    parser.add_argument(
+    group = parser.add_mutually_exclusive_group(required=False)
+    group.add_argument(
         "--stats",
-        help="generate statistics based on how out of date packages are",
+        help="generate statistics and output to stdout",
         default=False,
         action="store_true",
     )
+    group.add_argument(
+        "--send",
+        help="generate statistics and send to datadog",
+        default=False,
+        action="store_true"
+    )
     args = parser.parse_args()
 
     packages = get_packages(args.package_manager)
-    if args.stats:
+    if args.stats or args.send:
         stats = get_package_stats(packages)
-        # NOTE: subtle detail: we're depending on Python 3's ordered dict to
-        # maintain deterministic ordering here (critical when using
-        # --no-labels).
-        for key, value in stats.items():
-            print(f"{key}: {value}")
+        if args.stats:
+            # NOTE: subtle detail: we're depending on Python 3's ordered dict to
+            # maintain deterministic ordering here (critical when using
+            # --no-labels).
```

Comment:
What is `--no-labels`?

Reply:
Ah forgot to mention removing this ability, thanks for catching this. In the existing scripts, the`--no-labels` option allowed printing either the package list or stats without labels, so output that didn't include the header row for the table `Behind Package Latest Version`, or output that removed the following labels for stats:
```
Outdated:
Multi-Major:
Major:
Minor:
Patch:
Exotic:
```
I didn't come across any use case for either of these scenarios, so decided not to support it for now. I felt like the comment was still worth keeping to note that the order is still dependent on Python 3's ordered dict, but can remove the reference to `--no-labels`.

Reply:
See 56b8aeb1258aa16e71dd73310e4136f9578ab461 for removal of this reference.